okhttp-tests/src/test/java/okhttp3/CipherSuiteTest.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (C) 2016 Google Inc.
```

Comment:
Where in AOSP is that? I thought that generally the copyright line was assigned to "The Android Open Source Project":
https://android.googlesource.com/platform/libcore/+/master/luni/src/main/java/java/net/AddressCache.java
just like Guava's copyright lines are for "The Guava authors".

Perhaps OkHttp's copyright line should refer to "The OkHttp authors" or it should include an AUTHORS file? My understanding is that the Copyright line identifies the author of the file but does not impact Square's or anyone else's ability to reuse this code under the Apache 2 license. Is that correct? If so, a copyright line of "Square, Inc" would be simply incorrect since you've already accepted contributions by many external contributors?

Is that correct / how much do you care / would a copyright line of either "Google Inc" or "The OkHttp authors" (perhaps with an AUTHORS file) be fine with you?